standards/core/rendezvous.md

```diff
@@ -1,74 +1,124 @@
 ---
 title: WAKU-RENDEZVOUS
 name: Waku Rendezvous discovery
-editor: Simon-Pierre Vivier <simvivier@status.im>
-contributors:
+editor: Prem Chaitanya Prathi <premprathi@proton.me>
+contributors: Simon-Pierre Vivier <simvivier@status.im>
 ---
 
 ## Abstract
+
 This document describes the goal,
-strategy and usage of the libp2p rendezvous protocol by Waku.
+strategy, usage, and changes to the libp2p rendezvous protocol by Waku.
 
 Rendezvous is one of the discovery methods that can be used by Waku.
 It supplements Discovery v5 and Waku peer exchange.
 
 ## Background and Rationale
-How do new nodes join the network is the question that discovery answers.
-Discovery must be fast, pertinent and resilient to attacks.
-Rendezvous is both fast and allow the discovery of relevant peers,
-although it design can be easily abused
-due to it's lack of protection against denial of service atacks.
-The properties of rendezvous complements well the slower but safer methods like Discv5.
-To contribute well, a Waku node must know a sufficient number of peers with
-a wide variety of capabilities.
-By using rendezvous in combination with
+
+Waku nodes need a discovery mechanism that is both rapid and robust against attacks. Rendezvous enables quick identification of relevant peers, complementing slower but more resilient approaches like Discv5. For healthy network participation, nodes must connect to a diverse set of peers. However, the rendezvous protocol is vulnerable to denial of service attacks and depends on centralized rendezvous points, so it is best used in conjunction with other discovery methods such as Discv5.
```

Comment:
Please use semantic line breaks to make editing/reviewing easier: https://sembr.org/